diagnostics/version.rs

```diff
@@ -0,0 +1,7 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+pub const JSON_API_VERSION: u64 = 1;
```

Comment:
Couldn't come up with a better way of configuring it. Maybe it's better to place it to constants, but I like how it's placed in this package.